src/modules/inventory/components/AddProductForm.tsx

```diff
@@ -1,118 +1,277 @@
+            </Box>
+
+            <Box className='itemInput' sx={{ '& > :not(style)': { m: 1, width: '18ch' } }}>
+                <TextField
+                    id="stockQuantity-input"
+                    label="stockQuantity"
+                    variant="outlined"
+                    type="number"
+                    error={!!errors.stockQuantity}
+                    helperText={errors.stockQuantity?.message}
+                    {...register("stockQuantity")}
+                />
+            </Box>
+
+            <Box className='itemInput' sx={{ '& > :not(style)': { m: 1, width: '18ch' } }}>
+                <TextField
+                    id="bussinesId-input"
+                    label="bussinesId"
+                    variant="outlined"
+                    error={!!errors.bussinesId}
+                    helperText={errors.bussinesId?.message}
+                    {...register("bussinesId")}
+                />
+            </Box>
+
+            <Box className='itemInput' sx={{ '& > :not(style)': { m: 1, width: '18ch' } }}>
+                <TextField
+                    id="componentStatus-input"
+                    label="componentStatus"
+                    variant="outlined"
+                    error={!!errors.componentStatus}
+                    helperText={errors.componentStatus?.message}
+                    {...register("componentStatus")}
+                />
+            </Box>
+
+            <Box className='itemInput' sx={{ '& > :not(style)': { m: 1, width: '18ch' } }}>
+                <TextField
+                    id="productComponents-input"
+                    label="productComponents"
+                    variant="outlined"
+                    error={!!errors.productComponents}
+                    helperText={errors.productComponents?.message}
+                    {...register("productComponents")}
+                />
+            </Box>
+
+            <Box className='itemInput' sx={{ '& > :not(style)': { m: 1, width: '18ch' } }}>
+                <input type="file" multiple onChange={handleImageChange} />
+                {errors.componentsImages && <p>{errors.componentsImages.message}</p>}
+            </Box>
+
+            <Box className='itemInput' sx={{ '& > :not(style)': { m: 1, width: '18ch' } }}>
+                <label>
+                    isActive
+                    <input type="checkbox" {...register("isActive")} />
+                </label>
+                {errors.isActive && <p>{errors.isActive.message}</p>}
+            </Box>
+
+            <Box className='itemInput' sx={{ '& > :not(style)': { m: 1, width: '18ch' } }}>
+                <label>
+                    isOnSale
+                    <input type="checkbox" {...register("isOnSale")} />
+                </label>
+                {errors.isOnSale && <p>{errors.isOnSale.message}</p>}
+            </Box>
+
+            {/* <Button variant="contained" color="success" onClick={() => navigate('/')}>בחר רכיבים</Button> */}
```

Comment:
נא למחוק אם אין צורך

Reply:
פה הייתה לי שאלה -
עכשיו לשדה productComponents נתנו למשתמש להקליד סטרינגים המופרדים בפסיקים ואז חלקנו למערך ושלחנו כך.
השאלה אם בכלל השדה הזה צריך להיות מורכב מרשימה של קומפוננטות הקיימות במערכת ואז נצטרך קודם לשלוף רשימה של קומפוננטות ומשם יבחר?

Reply:
כן, אני חושבת שככה זה צריך לעבוד

Reply:
אוקי כי בצד שרת הוא התקבל כמערך של סטרינגים לכן עשינו כך נשנה את be ו fe לעשות כך בהתאם.
בכלל היה לנו קצת קשה להבין איפה בא לידי ביטוי הקשר בין ה products ל components

Reply:
לדוגמה השדה componentImages הוא מערך של תמונות הקומפוננטות ? אם כן למה אנחנו מוסיפים את זה בהוספת מוצר והוא לא נשלף מתוך הקומפוננטות שנבחרים מתוך הרשימה המוצגת של components?